pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -85,14 +85,13 @@ private static void generateSystemMetrics(SimpleTextOutputStream stream, String
             for (int i = 0; i < metricFamily.samples.size(); i++) {
                 Sample sample = metricFamily.samples.get(i);
                 stream.write(sample.name);
-                stream.write("{cluster=\"").write(cluster).write("\",");
+                stream.write("{cluster=\"").write(cluster).write('"');
```

Comment:
should we add any test to validate syntax of such outputs because it might be hard to catch such things in review and easy to make typo.??

Reply:
Yes, in the previous commit I did add a parser for the Prometheus text format to verify the metrics in the unit tests. Though, the parser is regex based and it's not validating 100% the format. In this PR I added the check to get the value of the tag, so that it was failing before the fix and passing after.